src/screens/planActivity/ShuffleButton.tsx

```diff
@@ -7,10 +7,12 @@ import { dimensions, palette } from '../../styles';
 
 interface Props {
   disabled?: boolean;
+  handleTouch?: () => void;
```

Comment:
I suggest to change naming convention to more device agnostic - no button clicks or touches, but button presses.